content/content-strategy/manage-content-requests/publish-content/tools.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: text/textblock
+---
```

Comment:
Missing `section` param. Please add it after `layout` along with the name of the heading.